packages/engine/src/simulation/package/state/packages/topology/mod.rs

```diff
@@ -30,7 +32,11 @@ impl PackageCreator for Creator {
         let topology = Topology {
             config: Arc::new(
                 TopologyConfig::create_from_globals(&config.sim.globals)
-                    .unwrap_or_else(|_| TopologyConfig::default()), // TODO OS - do we want to make a default if topology config is missing in Globals, or do we want to error
+                    .unwrap_or_else(|_| TopologyConfig::default()), /* TODO OS - do we want to
```

Comment:
Probably worth just moving this comment to the line above to avoid this weird formatting taking up a bunch of lines